addon/components/bs-accordion/item/_body.js

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
Why is the file prefixed with an underscore? Is it breaking if we have a `.js` file without a default export for a template-only component? Is this affecting all template-only components or is there something special about `<BsAccordion::Item::Body>`?

Reply:
> Is it breaking if we have a .js file without a default export for a template-only component? 

Exactly. 

> Is this affecting all template-only components

Yes.

Reply:
Thanks for explaining! Seems to be another reason to revisit the tools we use to generate our API docs.